docs/source-app/index.rst

```diff
@@ -216,17 +216,18 @@ Keep Learning
    :caption: How to...
 
    Access the App State <workflows/access_app_state/access_app_state>
-   Add a web user interface (UI) <workflows/add_web_ui/index>
    Add a web link  <workflows/add_web_link>
+   Add a web user interface (UI) <workflows/add_web_ui/index>
    Add encrypted secrets <glossary/secrets>
    Arrange app tabs <workflows/arrange_tabs/index>
+   Cache Work run calls  <workflows/run_work_once>
+   Customize your cloud compute <core_api/lightning_work/compute>
    Develop a Command Line Interface (CLI) <workflows/build_command_line_interface/index>
    Develop a Lightning App <workflows/build_lightning_app/index>
    Develop a Lightning Component <workflows/build_lightning_component/index>
    Develop a REST API <workflows/build_rest_api/index>
-   Cache Work run calls  <workflows/run_work_once>
-   Customize your cloud compute <core_api/lightning_work/compute>
    Extend an existing app <workflows/extend_app>
+   Mount Data from a Cloud Object Store to the Filesystem <workflows/mount_cloud_object_store>
```

Comment:
If we could shorten this a bit. I'm worried that the side bar navigation becomes to wide

Reply:
I went with `Mount Data From a Cloud Bucket`. There's lot's of type of data on the cloud, so it might be helpful to indicate what type of data we are talking about

Reply:
This is still really long.
The title should be based on the user motivation- they need to read data from cloud.
In the guide we should explain the possible solutions- currently only mounting an AWS bucket but it doesn't have to be in the title